docs/application/dotnet/guides/applications/service_application.md

```diff
@@ -0,0 +1,239 @@
+| Media                          | Playing audio, recording, and outputting streaming video in the background | [Tizen.Multimedia](/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.html) | `media`                                  |
+| Download                       | Downloading data with the classes and methods of the Tizen.Content.Download namespace | [Tizen.Content.Download](/application/dotnet/api/TizenFX/latest/api/Tizen.Content.Download.html) | `download`                               |
+| Background network             | Processing general network operations in the background (such as sync-manager, IM, and VOIP) | [Tizen.Account.SyncManager](/application/dotnet/api/TizenFX/latest/api/Tizen.Account.SyncManager.html) | `background-network`                     |
+| Location                       | Processing location data in the background | [Tizen.Location](/application/dotnet/api/TizenFX/latest/api/Tizen.Location.html) <br> [Tizen.Location.Geofence](/application/dotnet/api/TizenFX/latest/api/Tizen.Location.Geofence.html) <br> [Tizen.Maps](/application/dotnet/api/TizenFX/latest/api/Tizen.Maps.html) | `location`                               |
+| Sensor (context)               | Processing context data from the sensors, such as gesture | [Tizen.Sensor](/application/dotnet/api/TizenFX/latest/api/Tizen.Sensor.html) | `sensor`                                 |
+| IoT Communication/Connectivity | Communicating between external devices in the background (such as Wi-Fi and Bluetooth) | [Tizen.Network.WiFi](/application/dotnet/api/TizenFX/latest/api/Tizen.Network.WiFi.html) <br> [Tizen.Network.Bluetooth](/application/dotnet/api/TizenFX/latest/api/Tizen.Network.Bluetooth.html) | `iot-communication`                      |
+
+
+> [!NOTE]
+> Since Tizen 4.0, even if the background network category is declared, the running application stops if the network is not connected.
+
+### Describing background category
+
+An application with a background running capability must declare the background category in its manifest file:
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns="http://tizen.org/ns/packages" api-version="2.4" package="org.tizen.test" version="1.0.0">
+   <ui-application appid="org.tizen.test" exec="text" type="capp" multiple="false" taskmanage="true" nodisplay="false">
+      <icon>rest.png</icon>
+      <label>rest</label>
+      <!--For API version 2.4 and higher-->
+      <background-category value="media"/>
+      <background-category value="download"/>
+      <background-category value="background-network"/>
+   </ui-application>
+   <service-application appid="org.tizen.test-service" exec="test-service" multiple="false" type="capp">
+      <background-category value="background-network"/>
+      <background-category value="location"/>
+   </service-application>
+</manifest>
+```
+
+> [!NOTE]
```

Comment:
Note in line 84: These are about Tizen version before Tizen 2.4, but Tizen .NET started from Tizen 4.0, so I'm not sure if these notes are used to .NET aswell. Please consider revising this.